.github/workflows/link_reminder.yml

```diff
@@ -0,0 +1,33 @@
+name: Link reminder
+
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches: ["main"]
+
+jobs:
+  check_issue_link:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: read
+      pull-requests: write
+    steps:
+      - name: Wait
+        run: sleep 5
```

Comment:
if you're only matching on the PR body provided by the event payload, then this wait is likely unnecessary (even though I did specify it in the task)